docs/HowToGuides/FirstPullRequest.md

```diff
@@ -120,21 +130,26 @@ show a helpful "Compare & Pull Request" button.
 
 ![Compare & Pull Request button in GitHub UI](/docs/GitHubCreatePRScreenshot.png)
 
+[draftPR]: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests
 [create a pull request]: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request#creating-the-pull-request
 
 ## Asking for code review
 
-If you had an active discussion with someone on how to implement your change,
-you can `@` mention them in the PR description and ask for code review.
-If you directly implemented the change without any guidance from anyone else,
-`@` mention someone from GitHub's suggested reviewers. If GitHub doesn't
-make any suggestions, ask the [Code Owner](https://github.com/apple/swift/blob/main/CODE_OWNERS.TXT) based on the
-component for your change. Please ask someone though! We don't want you to get
-stuck because you were not sure who to ask for code review.
-
-At the beginning, contributors are not able to run the continuous integration
-(CI) bot, which builds the project and runs tests. Please ask your code
-reviewer(s) to invoke the bot for you.
+Collaborators and code owners are automatically requested for review per the
+[CODEOWNERS](/.github/CODEOWNERS) file. If no one is automatically requested for
+review, @mention and ask a suggested person under **Reviewers** to review your
+changes. If no reviewer suggestions are shown either, please ask @AnthonyLatsis
```

Comment:
@shahmishal Is it possible to add some a group?

Reply:
@AnthonyLatsis Maybe instead of this we should point them to write a message to contributor group on forums for now?

Reply:
We can create a group for triage team, which can be used here.

Reply:
I felt like that would be the next most sensible thing to do. "Next" because "go do a bunch of steps including a potential sign-up to contact the Contributor *Experience* group" 🥲. On the other hand, this fallback path is more or less cold.

Reply:
It’s possible to use github account there.

Reply:
>We can create a group for triage team, which can be used here. 
I take this back, we can't have a group with non-apple org contributors.

Reply:
Alright, let’s go with DMing the workgroup on the forums then.